### DISCUSSION TITLE
Surrounding by underscores in URL recognize to be as italic

### DISCUSSION THREAD

**Discussion by kay-adamof** [remarkjs/remark#1234] [Q&A] [answered]
My X(formerly twitter) account uses two underscores like this:

https://twitter.com/_adamof_/status/1681838181556318208

In this case, the URL doesn't recognize as URL properly while it makes the URL  act as just a string including italic body "_adamof_".

Does anyone know how to avert this behavior?

**Comment by wooorm** [comment]
Please provide some more info about what you are using and how :)

**Reply by kay-adamof** [reply]
Thank you for replying me. After sending my post, I realized my issue is relating to [the remark-embedder/core plugin](https://github.com/remark-embedder/core), not to "remarkjs". I'll repost the issue to that repository.

**Reply by wooorm** [reply, marked as answer]
could also be that you are not using `remark-gfm` — it adds support for “raw” URLs as links. It’s not standard markdown.

**Reply by kay-adamof** [reply]
Oh no ... . I'd like to send the best appreciation to you. You answer completely fixed my issue and save me a tons of times. Thank you, genius.`remark-gfm` was the key.
